server/controllers/controller.boardgames.js

```diff
@@ -0,0 +1,8 @@
+const { selectCategories } = require('../models/model.boardgames')
+
+exports.getCategories = (req, res) => {
+    selectCategories()
+    .then((categories)=>{
+        res.status(200).send(categories);
```

Comment:
I would send categories as a value on an object, then in your test you have a specified named key that represents the data you are sending over rather than just 'body'.